src/build_tools/hooks_hydra.py

```diff
@@ -437,24 +436,25 @@ def post_build_and_install_loop_hook(ecs_with_res):
     fancylogger.logToScreen(True, stdout=True)
     fancylogger.setLogLevelInfo()
 
-    do_lmod_cache = True
+    # by default we do not run the lmod cache
+    # only vsc10001 sets BUILD_TOOLS_RUN_LMOD_CACHE to 1 (via submit_build.py) to enable it
+    build_tools_lmod = os.getenv('BUILD_TOOLS_RUN_LMOD_CACHE', '0') == '1'
```

Comment:
```suggestion
    build_tools_lmod = os.getenv('BUILD_TOOLS_RUN_LMOD_CACHE', '0') != '0'
```
So any value that is not `0` is good ?

Reply:
i expanded to `.lower() not in ('0', 'false', 'off', 'no')`